README.md

```diff
@@ -3,6 +3,47 @@
 
 This is where we make API calls from users viewing and making changes to the State Benefits section of their income tax return.
 
+## Running the service locally
+
+You will need to have the following:
+- Installed [MondoDB](https://docs.mongodb.com/manual/installation/)
+- Installed/configured [service manager](https://github.com/hmrc/service-manager).
+
+The service manager profile for this service is:
+
+    sm --start INCOME_TAX_STATE_BENEFITS
+
+Run the following command to start the remaining services locally:
+
+    sudo mongod (If not already running)
+    sm --start INCOME_TAX_SUBMISSION_ALL -r
+
+This service runs on port: `localhost:9377`
+
+### Downstream services
+All State Benefits data is retrieved / updated via the downstream system.
+- IF (Integration Framework)
+
+### State Benefits Sources (HMRC-Held and Customer Data)
+State Benefits data can come from different sources: HMRC-Held and Customer. HMRC-Held data is state benefits data that HMRC have for the user within the tax year, prior to any updates made by the user. The state benefits data displayed in-year is HMRC-Held.
+
+Customer data is provided by the user. At the end of the tax year, users can view any existing state benefits data and make changes (create, update and delete).
+
+Examples can be found here in the [income-tax-submission-stub](https://github.com/hmrc/income-tax-submission-stub/blob/main/app/models/StateBenefitsUsers.scala)
```

Comment:
Probably you should say something like:
...
Examples of prior user data can be found ...
...

Reply:
Changed text